src/components/Form/File/index.js

```diff
@@ -0,0 +1,92 @@
+
+export default class File extends Component {
+   static propTypes = {
+      accepts: React.PropTypes.array,
+      height: React.PropTypes.string,
+      label: React.PropTypes.string,
+      maxFileSize: React.PropTypes.number,
+      name: React.PropTypes.string,
+      onChange: React.PropTypes.func,
+      style: React.PropTypes.object,
+      type: React.PropTypes.string,
+      width: React.PropTypes.string
+   }
+
+   constructor(props, context) {
+      super(props, context)
+
+      this.state = {
+         file: null,
+         src: ''
+      }
+   }
+
+   componentWillReceiveProps(props) {
+      this.setState({ src: props.src })
+   }
+
+   handleChange(files) {
+      this.setState({ file: files[0] }, () => {
+         this.props.onChange(null, this.state.file)
+      })
+   }
+
+   render() {
+      let dropzone
+      if (this.props.type === 'image') {
+         dropzone = <img className={formStyles.filesImage} src={imagePlaceholder} alt="placeholder" />
+         if (this.state.src) {
+            dropzone = <img className={formStyles.filesImage} src={this.state.src} alt="Avatar" />
+         }
+         if (this.state.file) {
+            dropzone = <img className={formStyles.filesImage} src={this.state.file.preview.url} alt="Avatar" />
+         }
+      } else {
+         dropzone = <div>Drop a file or click here to upload.</div>
+         if (this.state.file) {
+            dropzone = (
+               <div>
+                  <b>{this.state.file.name}</b> - {this.state.file.sizeReadable}
+               </div>
+            )
+         }
+      }
+
+      const style = {}
+      if (this.props.height) style.height = this.props.height
```

Comment:
Don't think you need the `if` statement here. Because `this.props` is guaranteed to be an object, the above code is practically equivalent to:

```js
const style = {
   height: props.height,
   width: props.width
}
```

If `props.height` were undefined, then `style.height` would also be undefined.

More importantly, it doesn't seem as though the `style` object is used, although I could be incredibly stupid. I do see that you use `this.props.style` on line 86, which seems to be separate.